src/js/components/Meter/Circle.js

```diff
@@ -13,12 +13,17 @@ const Circle = forwardRef((props, ref) => {
   const theme = useContext(ThemeContext);
   const width =
     size === 'full' ? 288 : parseMetricToNum(theme.global.size[size] || size);
-  const height =
-    type === 'pie'
-      ? width / 2
-      : parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  let height;
+  if (type === 'pie') {
+    height = width / 2;
+  } else if (type === 'semicircle') {
+    height =
+      2 * parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  } else
+    height = parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
   const mid = width / 2;
```

Comment:
How about we call this "centerX" and add a "centerY" and then use that as the first two arguments to arcCommands()?

Reply:
Im not sure if I follow why we would need `centerX` and `centerY`
`width/2` is passed in as the first 2 parameters for the arch

Reply:
I was trying to be more explicit to make it clear what we were using. For instance, the first two arguments to arcCommands are the center x,y coordinates for the arc. Using "mid" sometimes, but not always, makes the code a bit harder to follow.

Reply:
oh okay I see what you were trying to do I was exactly sure since we were using mid for both but yes I can add this so its more explicit what were passing in